### ISSUE TITLE
keydb 5.3.3 OS X Build Break

### ISSUE THREAD

**Issue by chenrui333** [JohnSully/KeyDB#160]
**Describe the bug**

relates to https://github.com/Homebrew/homebrew-core/pull/52164

** Log Files **
build log, https://github.com/Homebrew/homebrew-core/pull/52164/checks?check_run_id=536351908
```
In file included from fastlock.cpp:65:
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/ucontext.h:43:2: error: The deprecated ucontext routines require _XOPEN_SOURCE to be defined
#error The deprecated ucontext routines require _XOPEN_SOURCE to be defined
 ^
fastlock.cpp:180:5: error: use of undeclared identifier 'getcontext'
    getcontext(&ctxt);
    ^
2 errors generated.
```

Please copy everything after this line.

**To Reproduce**

- unzip v5.3.3.tar.gz
- make install

**Comment by JohnSully** [comment]
Simple fix, I updated the release tag to 98da479d

**Comment by chenrui333** [comment]
@JohnSully Thanks, it works for me.
